eZ/Publish/Core/Repository/ObjectStateService.php

```diff
@@ -615,12 +611,16 @@ protected function buildDomainObjectStateGroupObject(
      * @param string $identifier
      * @param string $defaultLanguageCode
      * @param string[] $names
-     * @param string[] $descriptions
+     * @param string[]|null $descriptions
      *
      * @return \eZ\Publish\SPI\Persistence\Content\ObjectState\InputStruct
      */
-    protected function buildCreateInputStruct($identifier, $defaultLanguageCode, $names, $descriptions)
-    {
+    protected function buildCreateInputStruct(
+        string $identifier,
+        string $defaultLanguageCode,
+        array $names,
+        ?array $descriptions
+    ): InputStruct {
         if (!is_string($identifier) || empty($identifier)) {
```

Comment:
we have `string $identifier` so `if (!is_string($identifier)` is not needed

Reply:
same in lines below

Reply:
also `is_array` check in https://github.com/ezsystems/ezpublish-kernel/pull/2728/files#diff-4ee5b7b94895992b9952ce0e423aa875R652 and https://github.com/ezsystems/ezpublish-kernel/pull/2728/files#diff-4ee5b7b94895992b9952ce0e423aa875R632